### DISCUSSION TITLE
Save posts when anonymous browsing

### DISCUSSION THREAD

**Discussion by Yuseldin** [Docile-Alligator/Infinity-For-Reddit#1394] [Ideas]
### Checklist

- [X] I have used the search function for [open](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/ideas) **and** [closed](https://github.com/Docile-Alligator/Infinity-For-Reddit/discussions/categories/completed-ideas) ideas to see if someone else has already submitted the same request.
- [X] I will describe the feature with as much detail as possible.

### Description

Is there a way to be able to save posts when browsing anonymously? 
I see that there's the ability to create multireddits and subscribe to subreddits without an account.
When saving posts it asks to login, same as when upvoting but upvoting would affect the count of upvotes so I can see why this wouldn't work, but with saved posts I don't think it affects if you have an account or not so it would be nice to have a local way to save posts.

### Additional context

Right now you have to login to your account. It'd be nice if there was a way to save posts locally. 

### How important is this feature?

Nice to have

**Comment by DecemberGiraffe** [comment]
Would also be interested in seeing this feature!
